MetaMask/metamask-mobile
# feat: enable support to Blockaid on custom networks on mobile

## **Description**



This is a follow up to https://github.com/MetaMask/metamask-mobile/pull/7547. Once this PR is merged, we'll be able to support additional networks on extension and we should enable the feature on them.

We should enabled it on the following networks:

Ethereum Mainnet (already done)
BNB chain
Polygon
Arbitrum
Optimism
Avalanche
Linea

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1693

## **Manual testing steps**

1. Go to Settings >> Network
2. Add custom networks
3. Go to Settings >> Experimental
4. See that blockaid toggle is only enabled for supported networks

## **Screenshots/Recordings**



### **Before**



### **After**

http://recordit.co/jC95wVGXEi



https://github.com/MetaMask/metamask-mobile/assets/29962968/dfcb4d8e-edb1-4698-8d6f-e1a44c15597e




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
